src/libpriv/rpmostree-core.c

```diff
@@ -1060,22 +1060,27 @@ rpmostree_hif_add_checksum_goal (GChecksum *checksum,
 {
```

Comment:
I can't comment any higher than this, but could you update the comment block above as well?